docs/docs/realtime.md

```diff
@@ -0,0 +1,689 @@
+
+Subscriptions are part of the GraphQL specification, whereas live queries are not.
+
+There are times where subscriptions are well-suited for a realtime problem — and in some cases live queries may be a better fit. Later we’ll explore the pros and cons of each approach and how best to decide that to use and when.
+
+### Defer and Stream
+
+[Stream and defer](https://the-guild.dev/graphql/yoga-server/docs/features/defer-stream) are directives that allow you to improve latency for clients by sending data the most important data as soon as it's ready.
+
+As applications grow, the GraphQL operation documents can get bigger.. The server will only send the response back once all the data requested in the query is ready. However, not all requested data is of equal importance, and the client may not need all of the data at once.
+
+#### Using Defer
+
+The `@defer`` directive allows you to post-pone the delivery of one or more (slow) fields grouped in an inlined or spread fragment.
+
+#### Using Stream
+
+The '@stream' directive allows you to stream the individual items of a field of the list type as the items are available.
+
+:::info
+The `@stream` directive is currently **not** supported by Apollo GraphQL client.
+:::
+
+## Features
+
+RedwoodJS Realtime handles the hard parts of a GraphQL Realtime implementation by automatically:
+
+- allowing GraphQL Subscription operations to be handled
+- merging in your subscriptions types and mapping their handler functions (subscribe, and resolve) to your GraphQL schema letting you keep your subscription logic organized and apart from services (your subscription my use a service to respond to an event)
+- authenticating subscription requests using the same `@requireAuth` directives already protecting other queries and mutations (or you can implement your own validator directive)
+- adding in the `@live` query directive to your GraphQL schema and setting up the `useLiveQuery` envelop plugin to handle requests, invalidation, and managing the storage mechanism needed
+- creating and configuring in-memory and persisted Redis stores uses by the PubSub transport for subscriptions and Live Queries (and letting you switch between them in development and production)
+- placing the pubSub transport and stores into the GraphQL context so you can use them in services, subscription resolvers, or elsewhere (like a webhook, function, or job) to publish an event or invalidate data
+- typing you subscription channel event payloads
+- support `@defer` and `@stream` directives
+
+It provides a first-class developer experience for real-time updates with GraphQL so you can easily
+
+- respond to an event (e.g. NewPost, NewUserNotification)
+- respond to a data change (e.g. Post 123's title updated)
+
+and have the latest data reflected in your app.
+
+Lastly, the Redwood CLI has commands to
+
+- generate a boilerplate implementation and sample code needed to create your custom
+    - subscriptions
+    - live Queries
```

Comment:
Thoughts on having this just be one level. E.g. don't break the sentence at 'to' and have it flow until 'custom' and then break into a list. Just since there is only one top level point